src/unicms_template_unical/templates/blocks/cdslist.html

```diff
@@ -0,0 +1,303 @@
+                                            <label for="department_id_input">Department Id</label>
+                                            <div class="w-30 mt-5">
+                                                <input type="text" id="department_id_input" name="department_id_input"
+                                                    class="form-control">
+                                            </div>
+                                        </div>
+                                    </div>
+                                    <div class="col-sm">
+                                        <div class="bootstrap-select-wrapper">
+                                            <label for="department_name_input">Department Name</label>
+                                            <div class="w-30 mt-5">
+                                                <input type="text" id="department_name_input"
+                                                    name="department_name_input" class="form-control">
+                                            </div>
+                                        </div>
+                                    </div>
+                                </div>
+                            </div>
+
+                            <!-- Course Type & Course Class Id & Course Class Name -->
+                            <div class="form-group">
+                                <div class="row">
+                                    <div class="col-sm">
+                                        <div class="bootstrap-select-wrapper">
+                                            <label for="course_type_input">Course Type</label>
+                                            <div class="w-30 mt-5">
+                                                <input type="text" id="course_type_input" name="course_type_input"
+                                                    class="form-control">
+                                            </div>
+                                        </div>
+                                    </div>
+                                    <div class="col-sm">
+                                        <div class="bootstrap-select-wrapper">
+                                            <label for="course_class_id_input">Course Class Id</label>
+                                            <div class="w-30 mt-5">
+                                                <input type="text" id="course_class_id_input"
+                                                    name="course_class_id_input" class="form-control">
+                                            </div>
+                                        </div>
+                                    </div>
+                                    <div class="col-sm">
+                                        <div class="bootstrap-select-wrapper">
+                                            <label for="course_class_name_input">Course Class Name</label>
```

Comment:
Stessa cosa qui

"Cource Class Name"

non va nel template perché una costante, se dovessi usare questo template per un altro scopo come faresti?

qui devi rappresentare il contenuto delle variabili, non copia/incolla nomi e valori dentro

i diversi col devono essere il risultato di un for, non li inserisci ad uno ad uno a mano così, è un template!